### PR TITLE
Missing Lat and Long for some cities.

### PR DESCRIPTION
**Pull request by esbanarango**
Addition of the missing coords.

### CODE REVIEW

File: lib/geolombia/fixtures/cities.csv [hunk 1]
```diff
@@ -320,7 +320,7 @@
 70233,El Roble,70,Sucre,9.1,-75.2
 25260,El Rosal,25,Cundinamarca,4.85,-74.2666667
 52256,El Rosario,52,Nariño,1.8333333,-77.3333333
-52258,El Tablón de Gómez,52,Nariño,,
+52258,El Tablón de Gómez,52,Nariño,5.982827,-76.304169
```
Comment (mejibyte) [inline lib/geolombia/fixtures/cities.csv]: Looks like these are the same coordinates that you put in "El Carmen de Atrato, Chocó". Could you update it here?

Comment (esbanarango) [inline lib/geolombia/fixtures/cities.csv]: Fixed.

**Comment by esbanarango** [comment]
@andmej Any chance to check this? Thanks.

**Comment by mejibyte** [comment]
Merged. Thanks!

**Pull request merged**